tests/async/test_tracing.py

```diff
@@ -28,6 +28,7 @@ async def test_browser_context_output_trace(
     page = await context.new_page()
     await page.goto(server.PREFIX + "/grid.html")
     await context.tracing.stop()
+    await page.wait_for_timeout(100)
```

Comment:
Do we have this upstream? Perhaps make it 1000?

Reply:
I tried to repro it upstream, but wasn't successful with it, will investigate further in a follow-up.